packages/backpack-core/config/webpack.config.js

```diff
@@ -115,7 +115,14 @@ module.exports = (options) => {
       new webpack.BannerPlugin({
         raw: true,
         entryOnly: false,
-        banner: `require('${require.resolve('source-map-support/register')}')`
+        banner: `require('${
+            // Is source-map-support installed as project dependency, or linked?
+            require.resolve('source-map-support').match(process.cwd())
```

Comment:
`match` finds in any place of the string, but `indexOf() === 0` ensures it's at the start.